### PR TITLE
TIMESTAMP support integer

### PR DESCRIPTION
**Pull request by veezhang**
fix: TIMESTAMP support integer
issue 229

## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -17,6 +17,10 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+var (
+	reTimestampInteger = regexp.MustCompile(`^(0[xX][0-9a-fA-F]+|0[0-7]+|\d+)$`)
```
Comment (yixinglu) [inline pkg/config/config.go:21]: `0o` prefix represent the octal number

Comment (veezhang) [reply pkg/config/config.go:21]: Yes, I know. And I've tried.

```
(root@nebula) [importer_test_v2]> RETURN TIMESTAMP(01234)
+----------------+
| timestamp(668) |
+----------------+
| 668            |
+----------------+
```

Comment (yixinglu) [reply pkg/config/config.go:21]: ACK.

**Review by yixinglu — APPROVED** [review]

**Comment by veezhang** [comment]
fix #229

**Pull request merged**